verilog/tools/ls/verilog-language-server_test.cc

```diff
@@ -576,6 +576,23 @@ std::string DefinitionRequest(absl::string_view file, int id, int line,
   return formattingrequest.dump();
 }
 
+// Performs assertions on textDocument/definition responses where single
+// definition is expected
+void CheckDefinitionResponseSingleDefinition(const json &response, int id,
+                                             int start_line,
```

Comment:
Nice, I like the improved readability and reduced code below by moving this verification to one function!

This sequence of 5 integers makes it hard to read and easy to swap out values. Maybe this could be two `verible::LineColumn` structs, so that we then can initialize them here

```
CheckDefinitionResponseSingleDefinition(response, 2, {.line = 1, .column = 9 }, { .line=1, .column=13}, "file://foo");

// or, if there is not enough space even the following would work as well (though above preferred)
CheckDefinitionResponseSingleDefinition(response, 2, {1, 9}, { 1, 13}, "file://foo");
```